### ISSUE TITLE
[suggestion] Shareable ESlint config

### ISSUE THREAD

**Issue by AntelaBrais** [CarlosVilasAlvarez/typescript-monorepo-template#3]
Having a shareable ESlint config as an autonomous library inside the monorepo would help opt-in/out in a more granular way. This could be helpful when integrating outside repos (as git submodules) with their own ESlint config already defined.

**Comment by CarlosVilasAlvarez** [comment]
I don't think that a shareable eslint would be useful, as you can extend the base one in each of the packages of the monorepo
